circle.yml

```diff
@@ -1,13 +1,19 @@
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I like to avoid shorthands when shell scripting. I particularly dislike programs which permit `-abc123` as shorthand for `-a -b -c 123` – how confusing!

Here I'd like to see `--location --output -` in place of `-o- -L`. Isn't `--output -` the default behaviour, though? `--location` should be sufficient.